docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,30 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.1]]
+=== 8.9.1
+
+[discrete]
+[[breaking-changes-8.9.1]]
+==== Breaking changes
+
+There are no breaking changes in 8.9.1.
+
+[discrete]
+[[enhancements-8.9.1]]
+==== Enhancements
+* Event correlation queries and rules can now detect missing events in EQL sequences.
+
+[discrete]
+[[bug-fixes-8.9.1]]
+==== Bug fixes 
+
+* Fixes a copy to clipboard bug that affected non-ECS fields ({pull}162883[#162883]).
+* Fixes rounding issues in the *Top alerts by* table on the Alerts page ({pull}162647[#162647]).
```

Comment:
I would just point out this is related to numbers/decimals because "rounding" can mean several things. 
```suggestion
* Fixes number rounding issues in the *Top alerts by* table on the Alerts page ({pull}162647[#162647]).
```